WatchFaceFormat/Weather/watchface/src/main/res/raw/watchface.xml

```diff
@@ -74,15 +74,11 @@
                 </Group>
                 <Group height="47" name="CurrentConditionIcon" width="47" x="215" y="0">
                   <Condition>
-                    <!--
-                      When using CONDITION, it is a requirement that all
-                      potential values are correctly handled. See:
-                      https://developer.android.com/docs/quality-guidelines/wear-app-quality
-                    -->
                     <Expressions>
+                      <Expression name="is_unknown">[WEATHER.CONDITION] == 0</Expression>
                       <Expression name="is_clear">[WEATHER.CONDITION] == 1</Expression>
                       <Expression name="is_cloudy">[WEATHER.CONDITION] == 2</Expression>
-                      <Expression name="is_fog">([WEATHER.CONDITION] == 3) || ([WEATHER.CONDITION] == 13)</Expression>
+                      <Expression name="is_mist">[WEATHER.CONDITION] == 13</Expression>
```

Comment:
reorder?

Reply:
@lucianbc is this something you can reorder on the source of truth?